integration/simulation/simulation.go

```diff
@@ -37,12 +37,14 @@ func (s *Simulation) Start() {
 	s.WaitForObscuroGenesis()
 
 	timer := time.Now()
+	fmt.Printf("Starting injection\n")
```

Comment:
Maybe use log.Info() for these injection start/stop logs? Or did you intend for it to just be on the console output?